packaging/systemd/sawtooth-poet.service

```diff
@@ -11,19 +11,18 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-# -----------------------------------------------------------------------------
+# ------------------------------------------------------------------------------
 
 [Unit]
-Description=Sawtooth PoET Engine
+Description=Sawtooth PoET Consensus Engine
 After=network.target
 
 [Service]
 User=sawtooth
 Group=sawtooth
-EnvironmentFile=-/etc/default/sawtooth-poet-engine
-ExecStart=/usr/bin/poet-engine $SAWTOOTH_POET_ENGINE_ARGS
+EnvironmentFile=-/etc/default/sawtooth-poet
+ExecStart=/usr/bin/sawtooth-poet $SAWTOOTH_POET_ARGS
```

Comment:
This should probably remain poet-engine?

Reply:
> DCO check is failing.

Will fix this